src/Sarif/Schemata/Sarif.schema.json

```diff
@@ -569,161 +559,133 @@
 
         "properties": 
         {
-          "description": "A dictionary consisting of a set of name/value pairs with arbitrary names. This allows tools to include information about the rule that is not explicitly specified in the SARIF format. The value within each name/value pair shall be a string, which may be the empty string. The value shall not be a dictionary or sub-object.",
+          "description": "Key/value pairs that provide additional information about the rule.",
           "type": "object",
           "default": {}
         },
 
-        "tags": 
-        {
-          "description": "A unique set of strings that provide additional information for the associated rule.",
+        "tags": {
+          "description": "A set of distinct strings that provide additional information about the rule.",
           "type": "array",
-          "items":
-          {
+          "items": {
             "type": "string",
-            "default": []
+            "default": [ ]
           }
         }
       },
       "required": ["id"]
     },
     "run": 
     {
-      "description": "A run object describes the invocation of the static analysis tool that produced the results specified in the containing runLog object. NOTE: The information in the run object makes it possible to precisely repeat a run of a static analysis tool, and to verify that the results reported in the log file were generated by an appropriate invocation of the tool.",
+      "description": "Describes a single run of a static analysis tool, and contains the output of that run.",
       "additionalProperties": false,
       "type": "object",
       "properties": {
-        "invocation": 
-        {
-          "description": "A string that describes any parameterization for the tool invocation. For command line tools this string may consist of the completely specified command line used to invoke the tool.",
+        "tool": {
+          "description": "Information about the tool or tool pipeline that generated the results in this log. A results log can only contain results produced by a single tool or tool pipeline. A results log can aggregate results from multiple tool log files, as long as context around the tool run (tool command-line arguments and the like) is identical for all aggregated files.",
```

Comment:
Why aren't you using run here? 'A run can only contain, blah'. If you like results log so much, that should be the name. :)